### PR TITLE
Add some vv tests using the docs as a template

### PR DESCRIPTION
**Pull request by jeanconn**
Added some vv tests using the docs as a template

### CODE REVIEW

**Comment by jeanconn** [comment]
Didn't realize when I started that the test setup stuff still had to go, so that's in there too.

File: mica/vv/tests/test_vv.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+def test_get_vv_files():
+    obsfiles = vv.get_vv_files(16504)
+    assert sorted(obsfiles)[-1] == os.path.abspath(os.path.join(common.MICA_ARCHIVE,
+                                                                'vv/16/16504_v01/vv_report.pkl'))
+
+def test_get_rms_data():
+    data = vv.get_rms_data()
+    dz_rms = data[(data['obsid'] == 16505) & (data['slot'] == 4) & (data['isdefault'] == 1)]['dz_rms'][0]
+    np.testing.assert_allclose(dz_rms, 0.047886185719034906)
```
Comment (taldcroft) [inline mica/vv/tests/test_vv.py]: Is there a reason to not use `assert np.allclose(..)`?  I think this gives better visibility in case of a failure.

Comment (jeanconn) [inline mica/vv/tests/test_vv.py]: That's fine.  I had just looked for a recent allclose example and copied it.

File: mica/vv/tests/test_vv.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    assert np.allclose(dz_rms, 0.047886185719034906)
+
+
+def test_get_vv():
+    obs = vv.get_vv(16504)
+    assert np.allclose(obs['slots']['7']['dz_rms'], 0.11610256063309182)
+
+
+def test_run_vv():
+    obi = process.get_arch_vv(2121)
```
Comment (jeanconn) [inline mica/vv/tests/test_vv.py]: This is the only interesting test, as get_arch_vv actually runs the mica.vv.Obi machinery.  The other tests are just fetching values from the tables.

**Comment by taldcroft** [comment]
Err, actually... fails Py3:
```
ska3-kadi$ python setup.py test
running test
running egg_info
writing dependency_links to mica.egg-info/dependency_links.txt
writing mica.egg-info/PKG-INFO
writing top-level names to mica.egg-info/top_level.txt
package init file 'mica/vv/tests/__init__.py' not found (or not a regular file)
reading manifest file 'mica.egg-info/SOURCES.txt'
writing manifest file 'mica.egg-info/SOURCES.txt'
running build_ext
=================================== test session starts ====================================
platform linux -- Python 3.5.2, pytest-3.0.5, py-1.4.31, pluggy-0.4.0
rootdir: /data/baffin/tom/git/mica, inifile: 
collected 9 items / 1 errors 

========================================== ERRORS ==========================================
________________________ ERROR collecting mica/vv/tests/test_vv.py _________________________
mica/vv/tests/test_vv.py:3: in <module>
    from .. import vv
E   SystemError: Parent module '' not loaded, cannot perform relative import
!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!
================================= 1 error in 14.55 seconds =================================
```

**Comment by jeanconn** [comment]
Well, at least that's something to fix.  I suppose I should have just made
my PR against master since I wasn't testing in Py3.
​

**Comment by jeanconn** [comment]
For Py3, I notice I'm missing Ska.Table in /proj/sot/ska3/test/ .  Do you want me to just expedite removing Ska.Table from mica?

**Pull request merged**